downloads/github-git-cheat-sheet.md

```diff
@@ -7,15 +7,15 @@ leadingpath: ../../
 ---
 
 {% capture colOne %}
-## Install Git
-GitHub provides desktop clients that include a graphical user interface for the most common repository actions and an automatically updating command line edition of Git for advanced scenarios.
+## Install
 
-### GitHub Desktop
-https://desktop.github.com/
+### GitHub for Windows
+https://windows.github.com
 
-Git distributions for Linux and POSIX systems are available on the official Git SCM website.
+### GitHub for Mac
+https://mac.github.com
```

Comment:
This was probably discussed when you were making the cheatsheet, but both the "for Windows" and "for Mac" links just point to desktop.github.com. Is this the intent? Should we just link to desktop.github.com?

Reply:
Good catch. I'll update that in this MD version...the PDF version is a bit more complicated to update, but we can open an issue for it.